app/views/taxYearOverview.scala.html

```diff
@@ -64,7 +64,8 @@ <h1 class="heading-xlarge">
 
             <table id="income-deductions-table">
                 <tr>
-                    <td>@messages("tax-year-overview.income")</td>
+                    <td id="income-row-title"><a href=@controllers.routes.IncomeSummaryController.showIncomeSummary(taxYear).url>
```

Comment:
Please do not add ids unless they are needed to for html/css functionality or on a dynamic table. As this table always has a set number of rows it is not required.